apps/node-fastify/src/app.ts

```diff
@@ -1,12 +1,18 @@
+// Needs to be imported here at the very beginning so that auto-instrumentation works for the imported modules
+import '../instrument.mjs';
+
 import fastify, { FastifyInstance, FastifyServerOptions } from 'fastify';
 import autoload from '@fastify/autoload';
 import path from 'node:path';
 import { OrbSync } from 'orb-sync-lib';
 import { getConfig } from './utils/config';
+import * as Sentry from '@sentry/node';
 
 export async function createApp(opts: FastifyServerOptions = {}): Promise<FastifyInstance> {
   const app = fastify(opts);
 
+  Sentry.setupFastifyErrorHandler(app);
```

Comment:
We should only set this up when Sentry is actually enabled (when SENTRY_DSN is set)

Reply:
Done